classes/article/ArticleGalleyDAO.inc.php

```diff
@@ -493,23 +530,26 @@ function getExportable($contextId, $pubIdType = null, $title = null, $author = n
 			FROM	submission_galleys g
 				JOIN submissions s ON (s.submission_id = g.submission_id)
 				LEFT JOIN published_submissions ps ON (ps.submission_id = g.submission_id)
+				JOIN submission_settings ss ON (ss.submission_id = ps.submission_id)
 				JOIN issues i ON (ps.issue_id = i.issue_id)
-				LEFT JOIN submission_files sf ON (g.file_id = sf.file_id)
-				LEFT JOIN submission_files nsf ON (nsf.file_id = g.file_id AND nsf.revision > sf.revision AND nsf.file_id IS NULL )
+				LEFT JOIN submission_galley_files sfg ON (sfg.galley_id = g.galley_id)
+				LEFT JOIN submission_files sf ON (sfg.file_id = sf.file_id)
+				LEFT JOIN submission_files nsf ON (nsf.file_id = sfg.file_id AND nsf.revision > sf.revision AND nsf.file_id IS NULL )
 				' . ($pubIdType != null?' LEFT JOIN submission_galley_settings gs ON (g.galley_id = gs.galley_id)':'')
 				. ($title != null?' LEFT JOIN submission_settings sst ON (s.submission_id = sst.submission_id)':'')
 				. ($author != null?' LEFT JOIN authors au ON (s.submission_id = au.submission_id)':'')
 				. ($pubIdSettingName != null?' LEFT JOIN submission_galley_settings gss ON (g.galley_id = gss.galley_id AND gss.setting_name = ?)':'') .'
 			WHERE
 				i.published = 1 AND s.context_id = ?
+				AND ss.setting_name = "datePublished" AND ss.submission_revision = 1
```

Comment:
In this case using single quotes would break the sql statement because datePublished needs to be a string.